### DISCUSSION TITLE
识别模型的标点符号该怎么训练

### DISCUSSION THREAD

**Discussion by TOMORINAONAO** [PaddlePaddle/PaddleOCR#13229] [Q&A]
最近在训练手写汉字的识别模型，在训练数据集的内容已经包含有标点符号的情况下，专门用手写字体合成了十万张纯英文字母+标点符号的混合数据。训练的模型文字识别效果还行，但标点的识别效果很差，差不多是必定会漏识别标点。想知道该怎么加强对标点符号的识别效果。

**Comment by SWHL** [comment]
应该是带有标点符号的数据集太少了，适当多增加一些
